src/components/pages/DAOcreator/index.tsx

```diff
@@ -198,11 +223,58 @@ class DAOcreator extends React.Component<Props, State> {
       }
     };
 
+    const PreviewDialog = () => (
```

Comment:
Let's make this dialog not be full screen, and be a bit larger than the other one so it overlays it.